cli/README.md

```diff
@@ -8,7 +17,36 @@ Then, you can use the CLI with the following command:
 ```shell
 java -cp "cli/build/libs/java-pfb-cli.jar" bio.terra.pfb.JavaPfbCommand <command>
```

Comment:
I'm not sure why I'm struggling with this -- I can't seem to specify the main class correctly. 
![image](https://github.com/DataBiosphere/java-pfb/assets/13254229/f71cb3e0-d443-4689-996d-77f76271762f)

Although, I'm not overly concerned with this -  I'd say the workflow you suggested below is even better (being able to run via gradle).

Reply:
If you don't need the .jar build to run while developing, I would get rid of the `jar` configuration completely and use the application plugin's `assembleDist` task. Once you've configured the `application` plugin, you can run 
```shell
./gradlew :cli:assembleDist
```
which creates`cli/build/distributions/cli-0.3.0.zip`. This is the artifact you'd publish to GitHub on a release.

To use this zip:
```
$ unzip ~/source/java-pfb/cli/build/distributions/cli-0.3.0.zip
Archive:  /Users/pshapiro/source/java-pfb/cli/build/distributions/cli-0.3.0.zip
   creating: cli-0.3.0/
   creating: cli-0.3.0/lib/
  inflating: cli-0.3.0/lib/cli-0.3.0.jar  
  inflating: cli-0.3.0/lib/picocli-4.7.4.jar  
  inflating: cli-0.3.0/lib/library-0.3.0.jar  
   creating: cli-0.3.0/bin/
  inflating: cli-0.3.0/bin/cli       
  inflating: cli-0.3.0/bin/cli.bat   
$ ./cli-0.3.0/bin/cli --version
pfb 0.3.0
$
```

Reply:
Oh cool! Why not both? I add the distribution to the GH release and notes in the README as to how to use both the jar and the distribution.

Reply:
Responding to your earlier comment, you're right that without `Main-Class` set you can't run `java xx.jar` directly. I thought `jar` would auto-configure based on `application` but you have to do it manually. This would work:

```
jar {
    manifest.attributes["Main-Class"] = application.mainClass
    ...
}
```
Once you do that you can run the .jar like this:
```
$ ./gradlew jar --quiet
$ java -jar cli/build/libs/java-pfb-cli-0.3.0.jar --version
pfb 0.3.0
$
```